docs/source/conf.py

```diff
@@ -23,7 +23,8 @@
 #sys.path.insert(0, os.path.abspath('.'))
 
 if os.environ.get('READTHEDOCS', ''):
-    # RTD doesn't use the Makefile, so re-run autogen_config.py here.
+    # RTD doesn't use the repo's Makefile to build docs. Run autogen_config.py
+    # here to generate rst files from ipynb files.
```

Comment:
`autogen_config.py` is actually generating the [config docs](http://nbconvert.readthedocs.org/en/latest/config_options.html), rather than converting ipynb to rst.